### ISSUE TITLE
Reproduced results differ from paper and training time inquiry

### ISSUE THREAD

**Issue by msjun23** [uzh-rpg/ssms_event_cameras#8]
Hi, thank you for publishing great work!

I have been trying to reproduce the results on the Gen1 dataset, but my findings are significantly different from those reported in the paper. Below are the details of my experiment setup and the results I obtained.

**Environment Configuration:**
```bash
GPU_IDS: [0,1]         
BATCH_SIZE_PER_GPU: 4  
TRAIN_WORKERS_PER_GPU: 12
EVAL_WORKERS_PER_GPU: 4
DATA_DIR: /root/data/gen1
MDL_CFG: base
```

I have used an A6000 40GB GPU (two units) instead of the A100 80GB (one unit) as mentioned in the GitHub repository. Since the total memory is the same, I divided the configurations accordingly to allocate the memory across the two GPUs.

I have not modified anything in RVT/config/experiment/gen1/default.yaml.

**Observed Results:**
During training, I noticed that the max_step was set to 400,000, causing the process to stop after one epoch. The total training time was approximately 67 hours (2 days and 19 hours).

Here are the validation metrics I obtained:
``` bash
- val/AP: 0.4406683404220787
- val/AP_50: 0.6706345137471073
- val/AP_75: 0.47589861083495194
- val/AP_L: 0.3979635967270493
- val/AP_M: 0.5222804973442176
- val/AP_S: 0.3784906607967766
```
These results are considerably lower than those reported in the paper.

**Questions:**
	1.	Why are the results different from the paper?
	2.	Is the training time normal? Is it expected to only run for 1 epoch?

Additionally, in the supplementary material, there are detection results images for the DSEC dataset. Could you please let me know if there are any available results or code for training on the DSEC dataset?

Thank you for your assistance.

**Comment by NikolaZubic** [comment]
@msjun23 Thanks for the questions. Total number of steps is important, which is 400k, meaning everything is set as expected.

1. We tested training on A100 GPU with 32-bit precision. I saw that there was sometimes problem with convergence if you train on different GPU, and with precision 16-bit it even diverges. In that sense, it is a little bit unstable, and that could be causing a problem for you. Regarding the inference, you have the trained models, and testing should yield the same results for arbitrary number of GPUs and GPU types. This has already been tested.

2. Regarding training time, I think it is okay assuming I needed a little bit less than 2 days on A100 for GEN1 dataset, and a little bit less than 3 days for 1Mpx dataset.
